src/defaultSelector.py

```diff
@@ -119,4 +325,32 @@ def rayIntersectsTriangleMollerTrumbore(self, K, P0, face: om.FaceHandle, mesh:
         # if t > e  and t < 1 - e:
         #    return  t
         # else:
-        #    return infinity
\ No newline at end of file
+        #    return infinity
+
+    def dmnsn_ray_box_intersection(self, optray: dmnsn_optimized_ray, box: dmnsn_aabb, t):
```

Comment:
Wasn't this very same code in `SubDivboxTree`? I would beg not to copy/paste the code.

Reply:
Done